### PR TITLE
fix(banner): replace button by a link - INNO-1535

### PR DESCRIPTION
**Pull request by emeryro**
# PR description

Replace call to action button by a standalone link in hero banner and page banner.

Note that the screenshot in usage page still displays a button...

### CODE REVIEW

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit 87a9f5621a9e6cf6672c103fe0d13ed8ab80673d

https://deploy-preview-1163--europa-component-library.netlify.com

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit 8951739cf109d4e13d6958bfa41bc9a17ad8d33b

https://deploy-preview-1163--europa-component-library.netlify.com

File: src/systems/ec/implementations/vanilla/packages/ec-component-hero-banner/ec-component-hero-banner.scss [hunk 1]
```diff
@@ -63,6 +65,16 @@
     margin-top: 0;
   }
 
+  .ecl-hero-banner__link {
+    font-weight: $ecl-font-weight-bold;
+    padding: $ecl-spacing-m $ecl-spacing-s;
+
+    &:hover,
+    &:focus {
+      color: $ecl-color-blue;
```
Comment (yhuard) [inline src/systems/ec/implementations/vanilla/packages/ec-component-hero-banner/ec-component-hero-banner.scss]: why do you change the colour here? is there an a11y issue somewhere?

File: src/systems/ec/implementations/vanilla/packages/ec-component-page-banner/ec-component-page-banner.scss [hunk 2]
```diff
@@ -65,6 +65,17 @@
...
   }
 
+  .ecl-page-banner__link {
+    font-weight: $ecl-font-weight-bold;
+    margin-top: $ecl-spacing-m;
+    padding: $ecl-spacing-m $ecl-spacing-s;
+
+    &:hover,
+    &:focus {
+      color: $ecl-color-blue;
```
Comment (yhuard) [inline src/systems/ec/implementations/vanilla/packages/ec-component-page-banner/ec-component-page-banner.scss]: same question as earlier, why do you change the colour?

**Review by yhuard — COMMENTED** [review]
nicely tackled, there shouldn't be any breaking change 👍 
I still have 1 question, nothing really blocking but please comment

**Review by yhuard — APPROVED** [review]

**Pull request merged**